frontend/components/ui/AgentHeaderV1.tsx

```diff
@@ -0,0 +1,26 @@
+import { LeftOutlined } from '@ant-design/icons';
```

Comment:
isn't it a bit misleading to call it AgentHeader? Think we can have a more generic name for this

Reply:
this component looks like just a BackButton